src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -286,3 +280,28 @@
       (ringutils/wrap-exception-handling)
       (ringutils/wrap-with-puppet-version-header puppet-version)
       (ringutils/wrap-response-logging)))
+
+(schema/defn ^:always-validate
+  get-wrapped-handler :- IFn
+  [route-handler :- IFn
+   ca-settings :- ca/CaSettings
+   path :- schema/Str
+   authorization-fn :- IFn
+   puppet-version :- schema/Str]
+  ;; For backward compatibility, requests to the .../certificate_status* endpoint
+  ;; will be authorized by a client-whitelist, if one is configured for
+  ;; 'certificate_status'.  When we are able to drop support for
+  ;; client-whitelist authorization later on, we should be able to get rid of
+  ;; the 'wrap-with-trapperkeeper-or-client-whitelist-authorization' function
+  ;; and replace with a line chaining the handler into a call to
+  ;; 'authorization-fn'.
+  (let [whitelist-path (str path
+                            (if (not= \/ path) "/")
```

Comment:
This wasn't right.  I was intending to have this just add "/" if the `path` didn't already end with one.  I changed this to ...

~~~~clojure
(if (not= \/ (last path)) "/")
~~~~

... in c456582.